.github/workflows/tests.yml

```diff
@@ -0,0 +1,17 @@
+name: "Tests"
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '5.x.x'
```

Comment:
Should this not be 6.x.x?  we are using dotnet 6 on this project

Reply:
You are correct, thanks for picking it up.